hooks/post-merge.sh

```diff
@@ -0,0 +1,3 @@
+npx lerna run build:travis \
```

Comment:
@brandon-binary as far as i remember npx downloads the package every single time.
since everyone has `lerna` installed globally already, maybe we can remove npx here to speed it up?

Second thing, can we actually do a `git diff HEAD HEAD~1 --ls-files` to figure out if it's needed to rebuild?

Reply:
@aminroosta I just checked and it seems that npx only reinstall the packages only if something is missing, say you do `npx lerna` and if `lerna` is not installed, then it will install it, otherwise it will use the locally/globally installed one, or use a temporary package that is downloaded if passed in using the `--package` flag. I think its best to keep this as it doesn't bring any overhead and works if users have `lerna` installed globally or locally, as oppose to only working with a global install of `lerna`.

For the second thing, I added the `--since` flag that `lerna` supports, now it will only build if one of the packages is changed. I tried to be more specific so that it only check for changes in `deriv-components` and `deriv-shared` but that led to a bug where if `deriv-components` is built without `deriv-shared` being built before, it will break since `deriv-components` is dependant on `deriv-shared`, so right now, we just build everything like how we have in the package lock.

`git diff HEAD HEAD~1 --ls-files` is a great, thank you.